extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -375,7 +375,7 @@ private RunningDevService startContainer(DockerStatusBuildItem dockerStatusBuild
     private static boolean isKeycloakX(DockerImageName dockerImageName) {
         return capturedDevServicesConfiguration.keycloakXImage.isPresent()
                 ? capturedDevServicesConfiguration.keycloakXImage.get()
-                : !dockerImageName.getVersionPart().endsWith(KEYCLOAK_LEGACY_IMAGE_VERSION_PART);
+                : dockerImageName.getVersionPart().endsWith(KEYCLOAK_LEGACY_IMAGE_VERSION_PART);
```

Comment:
@iocanel Not sure it is a correct fix, we have to check if it is Quakus based distro (formerly known as KeycloakX) so as long as it does not end with `-legacy` it is KeycloakX, but now you are saying it is Keycloak X if it ends with `-legacy` - but `-legacy` identifies WildFly based distros

Reply:
Aparently, I got it wrong. I falsely assumed that the `X` part meant `legacy`.